### ISSUE TITLE
Legend display bug with Application Insights when summarized

### ISSUE THREAD

**Issue by bhozar** [grafana/grafana#21105]
**What happened**:
Copied a working query from AI into Grafana. The legend displays in an odd format combining Y-axis and Split On fields. 

![image](https://user-images.githubusercontent.com/1184220/70854737-d20ae800-1eb7-11ea-844c-6220c673a218.png)

![image](https://user-images.githubusercontent.com/1184220/70854775-607f6980-1eb8-11ea-9be6-c842ac49639a.png)

**What you expected to happen**:
Legend to just display the **type** field.

**How to reproduce it (as minimally and precisely as possible)**:
```
dependencies
| where $__timeFilter()
| where operation_SyntheticSource == ""
| summarize percentile(duration, 95) by bin(timestamp,30m), type
| order by timestamp asc
Set Y-axis as timestamp, X-axis as percentile_95 and Split On as type
```
**Anything else we need to know?**:
I attempted to extract the data in the type= section with regex, but had no luck with `| extend type2 = extract("{type=(.*)}", 1, type)`

**Environment**:
- Grafana version: 6.5.1
- Data source type & version: Azure Application Insights
- OS Grafana is installed on: Ubuntu 18.04
- User OS & Browser: Windows 10 & Chrome

**Comment by bhozar** [comment]
Looks like it isn't just when using percentile you get the Legend format displaying oddly.

![image](https://user-images.githubusercontent.com/1184220/70870847-72ccd680-1f90-11ea-8f5c-cdd3a6bdcf98.png)

**Comment by diegofmole** [comment]
Since we update to Grafana v6.5.3, the legends of our charts that take metrics from AzureInsigths appear oddly too. Any idea when this bug will be solved?

![image](https://user-images.githubusercontent.com/19187807/73087679-a533f280-3eb1-11ea-9bc0-74457d423d1d.png)

**Comment by kylebrandt** [comment]
Application insights has been reworked for 7.1 to be like Log Analytics.

The default display, if format as time series, will take numeric column names as the metric name, and string columns as labels (more info at https://grafana.com/docs/grafana/latest/features/datasources/azuremonitor/#time-series-queries).

For example:

![image](https://user-images.githubusercontent.com/1692624/87580758-fa550200-c6a5-11ea-9699-1c6f9557c49d.png)

For display, there is now also field over rides for the graph panel (was just table before), so for a example series can be changed to just display the value for a dimension:

![image](https://user-images.githubusercontent.com/1692624/87581936-b662fc80-c6a7-11ea-866d-d2daf60854dd.png)

(Note the overrides are still beta for the panel, so multiple series matching isn't available for 7.1)

So since a lot of change for this data source, going to close this and assume it is fixed in 7.1. If it is not working after 7.1 please do reopen the issue.
